accounts-db/src/ancient_append_vecs.rs

```diff
@@ -328,11 +357,20 @@ impl AccountsDb {
         sorted_slots: Vec<Slot>,
         can_randomly_shrink: bool,
     ) {
+        /*
+        // maybe we want this to be a different tuning parameter passed on cli?
+        let max_ancient_slots = if let Some(offset) = self.ancient_append_vec_offset {
+            offset.unsigned_abs().min(10_000) as usize
+        } else {
+            return;
+        };
+        */
+        let max_ancient_slots = 100_000; // this gives us approx 5M ideal size for testing against mnb now
+
         let tuning = PackedAncientStorageTuning {
-            // only allow 10k slots old enough to be ancient
-            max_ancient_slots: 10_000,
-            // re-combine/shrink 55% of the data savings this pass
-            percent_of_alive_shrunk_data: 55,
+            max_ancient_slots,
+            // don't re-pack anything just to shrink. Shrink will handle these old storages.
+            percent_of_alive_shrunk_data: 0,
```

Comment:
I think we can change this tuning parameter to 0 to stop re-packing shrinkable slots. This mixes warm and cold accounts, which is not good. Once shrink is taking care of ancient storages, the pack algorithm should not mix old and new. It should only mix new and small in order to hit the storage count limit.